### PR TITLE
Add index.md beginning-course (Vietnamese translation)

### PR DESCRIPTION
**Pull request by duythanhvn**
#2717 Hello,

I have translated **index.md** from beginning-course into Vietnamese.

However, in the part that I read and found unclear, I wanted to confirm whether this piece of information was not finished or the content was just like that.

https://amp.dev/documentation/courses/beginning-course/?level=advanced

In this excerpt: _Otherwise, AMP has much in common with other front-end frameworks - so the skills you'll acquire while_

Kindly regards,
Duy Thanh Doan

### CODE REVIEW

**Comment by duythanhvn** [comment]
@googlebot I signed it!

**Comment by duythanhvn** [comment]
@googlebot I signed it!

**Comment by duythanhvn** [comment]
@googlebot I signed it!

File: pages/content/amp-dev/documentation/courses/beginning-course/index@vi.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+$title: Chào mừng bạn đến với Cộng đồng AMP!
+$order: 1
+$titles:
+  teaser: Khoá học Cơ bản
+teaser:
+  icon: airplane
+  label: Bắt đầu ngay
+description: 'Hiểu về AMP và cách mà AMP có thể giúp đỡ. Tạo trang AMP đầu tiên của bạn ngay!'
+leveled: true
+---
```
Comment (CrystalOnScript) [inline pages/content/amp-dev/documentation/courses/beginning-course/index@vi.md]: Thanks so much! One small nit - everything should be removed from the front matter except the translated title :)

Comment (duythanhvn) [reply pages/content/amp-dev/documentation/courses/beginning-course/index@vi.md]: Hi @CrystalOnScript , I fixed it.

**Pull request merged**